ui/src/app/pages/login/account-creation/account-creation-options.tsx

```diff
@@ -59,6 +63,80 @@ export const AccountCreationOptions = {
     {label: 'Administrator', value: EducationalRole.ADMIN},
     {label: 'Other (free text)', value: EducationalRole.FREETEXT}
   ],
+  // END roles and affiliations for old (pre-verification) institutional affiliation
+  // BEGIN roles for verified institutional affiliation
```

Comment:
This is **_mostly_** a copy of the old-flow roles but **_not quite_**, so I'm making a clean break and deleting the old ones later, as planned here: https://docs.google.com/document/d/1qrTwj2-YH3-r_nAveQnY75L5GCLuaBwqfwSciiF4p3M/edit#bookmark=id.3ah876rsgvdu